assets/javascripts/app/LeafletMap.js

```diff
@@ -0,0 +1,36 @@
+export default class LeafletMap {
+  static get ROUTE_COLOR() {
+    return 'red';
```

Comment:
I would prefer the stylesheet's mountain red https://github.com/HomebyTwo/homebytwo/blob/master/assets/stylesheets/settings/_settings.scss#L17